deploy/charts/cert-manager/values.yaml

```
@@ -469,29 +469,29 @@ prometheus:
     # +docs:property
     endpointAdditionalProperties: {}
 
-  # Note: Enabling both PodMonitor and ServiceMonitor is mutually exclusive, enabling both will result in a error.
+  # Note that you cann enable both PodMonitor and ServiceMonitor as they are mutually mutually exclusive. Enabling both will result in a error.
```

suggestion: typo, extra word!

```suggestion
  # Note that you can not enable both PodMonitor and ServiceMonitor as they are mutually exclusive. Enabling both will result in a error.
```